indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -175,3 +206,9 @@ export default function EditableList() {
     </>
   );
 }
+
+EditableListDisplay.propTypes = {
+  editableList: PropTypes.array.isRequired,
```

Comment:
`PropTypes.arrayOf(PropTypes.shape(...)).isRequired` (in the shape specify the props you are actually using).